demo/reset-counters.sh

```diff
@@ -7,10 +7,10 @@ source .env
 
 # This script assumes that port forwarding has already been established.
 # See ./scripts/port-forward-all.sh to enable fort forwarding for Bookstore demo.
-curl -I -X GET http://localhost:8080/reset &
-curl -I -X GET http://localhost:8081/reset &
-curl -I -X GET http://localhost:8082/reset &
-curl -I -X GET http://localhost:8083/reset &
+curl -I -X GET http://127.0.0.1:8080/reset &
```

Comment:
Did this also not work for you as localhost?

Reply:
I just cmd+f changed them all. I'll leave them in the next PR